components/authz-service/storage/v2/postgres/postgres.go

```diff
@@ -792,6 +874,11 @@ func (p *pg) UpdateRole(ctx context.Context, role *v2.Role) (*v2.Role, error) {
 		}
 	}
 
+	err = p.notifyPolicyChange(ctx, tx)
+	if err != nil {
+		return nil, p.processError(err)
+	}
+
```

Comment:
⚠️ I didn't verify that every function that needs this now has this. But I guess your test change does cover this, after all.

Reply:
i did my best to line the up with what the interceptor does, but please do double check